code/datums/elements/newton.dm

```diff
@@ -0,0 +1,54 @@
+/datum/element/newton/Attach(datum/target)
+	. = ..()
+	if(!ismob(target))
+		return ELEMENT_INCOMPATIBLE
+
+	RegisterSignal(target, COMSIG_HUMAN_ITEM_THROW, .proc/throw_item)
+	RegisterSignal(target, COMSIG_MOB_EXTINGUISHER_USE, .proc/exinguish)
+	RegisterSignal(target, list(COMSIG_HUMAN_GUN_FIRED, COMSIG_HUMAN_GUN_AUTOFIRED), .proc/fire_gun)
+
+/datum/element/newton/Detach(datum/target, force)
+	. = ..()
+	UnregisterSignal(target, list(
+		COMSIG_MOB_EXTINGUISHER_USE, 
+		COMSIG_HUMAN_ITEM_THROW, 
+		COMSIG_HUMAN_GUN_FIRED, 
+		COMSIG_HUMAN_GUN_AUTOFIRED))
+
+/datum/element/newton/proc/exinguish(datum/source, atom/target_atom, force)
+	var/mob/M = source
+
+	propel(M, get_dir(M, target_atom), force)
+
+/datum/element/newton/proc/throw_item(datum/source, atom/target)
+	var/mob/M = source
+
+	propel(M, get_dir(M, target), 0)
+
+/datum/element/newton/proc/fire_gun(datum/source, atom/target, obj/item/weapon/gun/gun, mob/living/user)
+	propel(user, get_dir(user, target), 0)
+
+/datum/element/newton/proc/propel(mob/M, direction, force)
+	if(!M.buckled || !isobj(M.buckled) || M.buckled.anchored)
+		Detach(M)
+
+	var/rep = CLAMP((4-force)*2, 0, 8)
+
+	move_chair(M.buckled, reverse_direction(direction), rep)
```

Comment:
This seems very strange to be adding to the mob whenever they sit on a chair, not only that you're assuming here that the chair exists. You should instead either make the element go on the chair and use a signal for when mobs buckle or make the element more generic.

If it *is* generic you probably want to make it support going on either side, and at that point you've recreated space drift.